pre-cache/copy-env.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+cwd="${cwd:-/opt/precache}"
```

Comment:
what is cwd here? shouldn't you use pwd?

Reply:
No, it is for unit testing:
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/04a3b444fc20580d28d41cd266452e0252d2435c/Makefile#L348